Library/Homebrew/test/cmd/commands_spec.rb

```diff
@@ -6,6 +6,12 @@
   it "prints a list of all available commands" do
     expect { brew "commands" }
       .to output(/Built-in commands/).to_stdout
+      .and not_to_output.to_stderr
+      .and be_a_success
+
+    expect { brew "commands", "--quiet" }
```

Comment:
This should be a separate test and should check that the output doesn't include `Built-in commands`.